onos-classic/templates/configmap-probe.yaml

```diff
@@ -13,6 +13,26 @@ data:
     #!/bin/bash
     set -e
     host=$(hostname -s)
-    config=$(curl -s http://$host:8181/onos/v1/cluster/$host --user onos:rocks)
-    echo $config
-    printf '%q' $config | grep -q "READY"
+    result=$(curl -s -f http://$host:8181/onos/v1/cluster/$host --user onos:rocks)
```

Comment:
you don't want to hardcode credentials

Reply:
This helm chart doesn't support changing login credentials anyway. It will be more meaningful if we implement that as well. I'd leave this to a separate patch.